kernel/malloc.c

```diff
@@ -0,0 +1,301 @@
+#include <limits.h>
```

Comment:
Copyright header is missing.

Which version of dlmalloc is this? Also should we pay tribute to the dlmalloc author?

Reply:
Fixed. I added comments that give tribute and point to the location of the license.

Reply:
Version is 2.8.6. Same as OE.